tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
This project is not built in typescript, so this file could be confusing, could you remove it if possible? I think we can use `node-ts` module to avoid compilation step altogether.

Reply:
File removed in favor of node-ts.